packages/lib-sourcify/src/lib/verification.ts

```diff
@@ -179,11 +179,11 @@ export async function verifyDeployed(
         generateCreationCborAuxdataPositions,
         recompiled.creationLinkReferences,
       );
-      if (match.runtimeMatch === 'partial') {
+      if (match.creationMatch === 'partial') {
         logDebug('Matched partial with creation tx', {
           chain: sourcifyChain.chainId,
           address,
-          runtimeMatch: match.runtimeMatch,
+          creationMatch: match.creationMatch,
           creatorTxHash,
         });
         match = await tryToFindPerfectMetadataAndMatch(
```

Comment:
Just FYI, I thought that also the `runtimeBytecode` passed to `tryToFindPerfectMetadataAndMatch` was wrong but `tryToFindPerfectMetadataAndMatch` use it only to extract the decoded auxdata.